test/integration/targets/ec2_ami/tasks/main.yml

```diff
@@ -118,6 +118,23 @@
 
     # ============================================================
 
+    - name: gather facts about the image created
+      ec2_ami_facts:
```

Comment:
Probably worth adding ec2_ami_facts to test/integration/targets/ec2_ami/aliases

Reply:
I am not sure how this works. Does "ec2_ami_facts" need to be on a separate line by itself?

What effect does it have? Would that make the `ec2_ami` target run on any commits to the `ec2_ami_facts` module?

Reply:
You're quite right on both points.